ubuntu1604/deps_spec.yaml

```diff
@@ -22,6 +23,20 @@ gcsDeps:
     versionRegex: "\\d{8,}"
     fileRegex: "^ubuntu1604/tar/\\d{8,}_ubuntu1604\\.tar\\.gz$"
     startsWith: "ubuntu1604/tar/"
+    fusMetadataBucket: "container-deps"
+    fusMetadataObject: "ubuntu1604/metadata/debs/metadata.yaml"
+    releasePolicies:
+      - tag: "default"
+        # Weekly release schedule at 5am every Monday.
```

Comment:
lets add a TODO to move this to montly (or even less periodicity) once we have tested it fully

Reply:
done